Gems/PhysX/Code/Tests/CharacterControllerTests.cpp

```diff
@@ -159,6 +159,115 @@ namespace PhysX
         }
     }
 
+    TEST_F(PhysXDefaultWorldTest, CharacterController_GamePlayComponenet_EntityFallsUnderGravity)
```

Comment:
nitpick - typo in Component (here and other places)

Reply:
Fixed